staging/src/kubevirt.io/api/core/v1/schema.go

```diff
@@ -739,6 +739,10 @@ type SEVPolicy struct {
 	// Defaults to false.
 	// +optional
 	EncryptedState *bool `json:"encryptedState,omitempty"`
+	// SEV-SNP is required.
+	// Defaults to false.
+	// +optional
+	SecureNestedPaging *bool `json:"secureNestedPaging,omitempty"`
```

Comment:
hi @ajcaldelas , refer to https://libvirt.org/formatdomain.html#launch-security, there are still several options to be configured for a snp-guest, e.g. hostData I mentioned in today's meeting, do you plan to support them in this PR? 

FYI, here is an example of launchsecurity config:
<domain>
  ...
  <launchSecurity type='sev-snp' authorKey='yes' vcek='no'>
    <cbitpos>47</cbitpos>
    <reducedPhysBits>1</reducedPhysBits>
    <policy>0x00030000</policy>
    <guestVisibleWorkarounds>...</guestVisibleWorkarounds>
    <idBlock>...</idBlock>
    <idAuth>...</idAuth>
    <hostData>...</hostData>
  </launchSecurity>
  ...
</domain>

Reply:
I believe this discussion can be closed as resolved.